src/config.rs

```diff
@@ -145,6 +145,7 @@ pub struct Config {
     max_redirects: u32,
     max_redirects_will_error: bool,
     redirect_auth_headers: RedirectAuthHeaders,
+    redirect_history: bool,
```

Comment:
Let's call this `save_redirect_history`, if we follow the logic of "verb + noun" it sounds a bit strange to "redirect history" (redirect it where?).